uninstall.sh

```diff
@@ -0,0 +1,326 @@
+  for word in $1; do
+    output "$word"
+  done
+  print_brake 30
+  echo ""
+}
+
+error() {
+  COLOR_RED='\033[0;31m'
+  COLOR_NC='\033[0m'
+
+  echo ""
+  echo -e "* ${COLOR_RED}ERROR${COLOR_NC}: $1"
+  echo ""
+}
+
+warning() {
+  COLOR_YELLOW='\033[1;33m'
+  COLOR_NC='\033[0m'
+  echo ""
+  echo -e "* ${COLOR_YELLOW}WARNING${COLOR_NC}: $1"
+  echo ""
+}
+
+summary() {
+  print_brake 30
+  output "Uninstall panel? $RM_PANEL"
+  output "Uninstall wings? $RM_WINGS"
+  print_brake 30
+}
+
+####### OS check funtions #######
+
+detect_distro() {
+  if [ -f /etc/os-release ]; then
+    # freedesktop.org and systemd
+    . /etc/os-release
+    OS=$(echo "$ID" | awk '{print tolower($0)}')
+    OS_VER=$VERSION_ID
+  elif type lsb_release >/dev/null 2>&1; then
+    # linuxbase.org
+    OS=$(lsb_release -si | awk '{print tolower($0)}')
+    OS_VER=$(lsb_release -sr)
+  elif [ -f /etc/lsb-release ]; then
+    # For some versions of Debian/Ubuntu without lsb_release command
+    . /etc/lsb-release
+    OS=$(echo "$DISTRIB_ID" | awk '{print tolower($0)}')
+    OS_VER=$DISTRIB_RELEASE
+  elif [ -f /etc/debian_version ]; then
+    # Older Debian/Ubuntu/etc.
+    OS="debian"
+    OS_VER=$(cat /etc/debian_version)
+  elif [ -f /etc/SuSe-release ]; then
+    # Older SuSE/etc.
+    OS="SuSE"
+    OS_VER="?"
+  elif [ -f /etc/redhat-release ]; then
+    # Older Red Hat, CentOS, etc.
+    OS="Red Hat/CentOS"
+    OS_VER="?"
+  else
+    # Fall back to uname, e.g. "Linux <version>", also works for BSD, etc.
+    OS=$(uname -s)
+    OS_VER=$(uname -r)
+  fi
+
+  OS=$(echo "$OS" | awk '{print tolower($0)}')
+  OS_VER_MAJOR=$(echo "$OS_VER" | cut -d. -f1)
+}
+
+check_os_comp() {
+  SUPPORTED=false
+  case "$OS" in
+  ubuntu)
+    [ "$OS_VER_MAJOR" == "18" ] && SUPPORTED=true
+    [ "$OS_VER_MAJOR" == "20" ] && SUPPORTED=true
+    ;;
+  debian)
+    [ "$OS_VER_MAJOR" == "9" ] && SUPPORTED=true
+    [ "$OS_VER_MAJOR" == "10" ] && SUPPORTED=true
+    ;;
+  centos)
+    [ "$OS_VER_MAJOR" == "7" ] && SUPPORTED=true
+    [ "$OS_VER_MAJOR" == "8" ] && SUPPORTED=true
+    ;;
+  esac
+
+  # exit if not supported
+  if [ "$SUPPORTED" == true ]; then
+    echo "* $OS $OS_VER is supported."
+  else
+    echo "* $OS $OS_VER is not supported"
+    error "Unsupported OS"
+    exit 1
+  fi
+}
+
+### Main uninstallation functions ###
+
+rm_panel_files() {
+  output "Removing panel files..."
+  rm -rf /var/www/pterodactyl /usr/local/bin/composer
+  [ "$OS" != "centos" ] && unlink /etc/nginx/sites-enabled/pterodactyl.conf
+  [ "$OS" != "centos" ] && rm -f /etc/nginx/sites-available/pterodactyl.conf
+  [ "$OS" == "centos" ] && rm -f /etc/nginx/conf.d/pterodactyl.conf
+  output "Succesfully removed panel files."
+}
+
+rm_wings_files() {
+  output "Removing wings files..."
+  rm -rf /etc/pterodactyl /usr/local/bin/wings /var/lib/pterodactyl
+  output "Succesfully removed wings files."
+}
+
+rm_services() {
+  output "Removing services..."
+  systemctl disable --now mariadb
```

Comment:
Should we really disable MariaDB? What if the user is using it for something else? I reckon we should just remove the database, but not stop the service itself.